code/datums/datum.dm

```diff
@@ -0,0 +1,14 @@
+/datum
```

Comment:
I'm not sure I like adding any more files to the gross mess that is code/datums/
Any particular reason this can't just go in the GC code?

Reply:
Parent types should generally get their own file and all their vars should be there; I'm not going to hunt down all the `/datum` vars and move them here in *this* PR though.

Reply:
That's going to be a fun one.. 😱

Reply:
Ah the never ending tug of war between maintainability and modularity. Having the base level defines in their own files makes things more modular but having them in one file makes things easier to find.

Reply:
I went ahead and moved the datum vars over to their correct file; wasn't nearly as many as I thought there would be.

Reply:
Okay, let me rephrase: Why put this here instead of code/modules/datum?

Reply:
This is where the vast majority of the other datums are stored that are generic and not tied with a particular module; typically you want the parent type kept close with the others--the base carbon/human/simple_animal/silicon types are defined close to where the other subtypes are.

The objects folder is structured the same way.

Don't really see any reason to really move it.